### DISCUSSION TITLE
Unable to render this definition

### DISCUSSION THREAD

**Discussion by stanko04** [jellydn/next-swagger-doc#1164] [Q&A]
### Describe the bug

I got this on my localhost:3000/api-doc page: 

![bug](https://github.com/user-attachments/assets/7817cdf6-5d3a-4cbd-b8e8-c90aa8fd70c5)

I have everything like it is in tutorial: 

My swagger.ts:
![image](https://github.com/user-attachments/assets/9b6b8311-f12b-470c-a087-f17c4cc10911)

react-swagger.tsx:
![image](https://github.com/user-attachments/assets/4df6aab3-b4df-48aa-858f-bfead2dbef06)

api-doc/page.tsx: 
![image](https://github.com/user-attachments/assets/232b82cf-9e95-4c06-b34f-0a351cd67fe9)

app/api/hello/route.ts: 
![image](https://github.com/user-attachments/assets/008f2b15-a1fb-414a-a72e-a551a79d0741)

And I got still HTTP 500 code, during rendering api-doc page. And got that page "Unable to render this definition...". 

I tried to with next 15, but also with next 14 it is not working. 
And also tried with react 19 and react 18 and still not working.

  
What am I missing during configuration ? 

### Reproduction

Reproduce steps in tutorial.

### System Info

```Shell
My package.json: 
` "dependencies": {
    "@joint/core": "^4.0.4",
    "@prisma/client": "^5.21.1",
    "antlr4ts": "^0.5.0-alpha.4",
    "bcryptjs": "^2.4.3",
    "class-variance-authority": "^0.7.0",
    "clsx": "^2.1.1",
    "lucide-react": "^0.453.0",
    "next": "15.0.0",
    "next-auth": "^5.0.0-beta.25",
    "next-swagger-doc": "^0.4.1",
    "react": "19.0.0-rc-65a56d0e-20241020",
    "react-dom": "19.0.0-rc-65a56d0e-20241020",
    "swagger-ui-react": "^5.17.2",
    "tailwind-merge": "^2.5.4",
    "tailwindcss-animate": "^1.0.7",
    "uuid": "^10.0.0",
    "openapi-types": "^12.1.3"
  },
  "devDependencies": {
    "@trivago/prettier-plugin-sort-imports": "^4.3.0",
    "@types/bcryptjs": "^2.4.6",
    "@types/node": "^20",
    "@types/react": "^18",
    "@types/react-dom": "^18",
    "@types/swagger-ui-react": "^4.18.3",
    "@types/uuid": "^10.0.0",
    "eslint": "^9.13.0",
    "eslint-config-next": "15.0.0",
    "postcss": "^8",
    "prettier": "^3.3.3",
    "prettier-plugin-tailwindcss": "^0.6.8",
    "prisma": "^5.21.1",
    "tailwindcss": "^3.4.1",
    "typescript": "^5"
  },`
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [x] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by jellydn** [comment]
Could you try with example app on this repo yet? @stanko04
